utils/build.ps1

```diff
@@ -1858,16 +1858,20 @@ function Build-Dispatch([Platform]$Platform, $Arch, [switch]$Test = $false) {
     $InstallPath = "$($Arch.SDKInstallRoot)\usr"
   }
 
-  Build-CMakeProject `
-    -Src $SourceCache\swift-corelibs-libdispatch `
-    -Bin (Get-TargetProjectBinaryCache $Arch Dispatch) `
-    -InstallTo $InstallPath `
-    -Arch $Arch `
-    -Platform $Platform `
-    -UseBuiltCompilers C,CXX,Swift `
-    -Defines @{
-      ENABLE_SWIFT = "YES";
-    }
+  Isolate-EnvVars {
+    $env:Path = "$(Get-CMarkBinaryCache $BuildArch)\src;$(Get-PinnedToolchainRuntime);${env:Path}"
```

Comment:
Can we not sink this into the `Build-CMakeProject`?

Reply:
Done